### ISSUE TITLE
Increase telethon version to avoid exceptions

### ISSUE THREAD

**Issue by wowkin2** [Kosat/telegram-messages-dump#15]
Got an issue with current using version 0.18.3
```
Traceback (most recent call last):
  File "...\telegram_messages_dump\telegram_dumper.py", line 80, in run
    chatObj = self._getChannel()
  File "...\telegram_messages_dump\telegram_dumper.py", line 173, in _getChannel
    dialogs = self.get_dialogs(limit=None)
  File "...\venv\lib\site-packages\telethon-0.18.3-py3.6.egg\telethon\telegram_client.py", line 619, in get_dialogs
    dialogs = UserList(self.iter_dialogs(*args, **kwargs))
  File "C:\Python36\lib\collections\__init__.py", line 1039, in __init__
    self.data = list(initlist)
  File "...\venv\lib\site-packages\telethon-0.18.3-py3.6.egg\telethon\telegram_client.py", line 600, in iter_dialogs
    yield Dialog(self, d, entities, messages)
  File "...\venv\lib\site-packages\telethon-0.18.3-py3.6.egg\telethon\tl\custom\dialog.py", line 69, in __init__
    self.draft = Draft(client, dialog.peer, dialog.draft)
  File "...\venv\lib\site-packages\telethon-0.18.3-py3.6.egg\telethon\tl\custom\draft.py", line 32, in __init__
    self._text = markdown.unparse(draft.message, draft.entities)
AttributeError: 'DraftMessageEmpty' object has no attribute 'message'
```
It is already investigated and fixed in next releases (the latest is `1.0.3`).
See discussion: https://github.com/LonamiWebs/Telethon/issues/844

**Comment by wowkin2** [comment]
Looks like bug was fixed in `0.19.1.6` (oldest that works for me, all the next are async)

**Comment by Kosat** [comment]
Ok. Thank you. I merged in your changes.
